static/app/components/replays/canvasSupportNotice.tsx

```diff
@@ -10,16 +10,22 @@ import {space} from 'sentry/styles/space';
 import {trackAnalytics} from 'sentry/utils/analytics';
 import useDismissAlert from 'sentry/utils/useDismissAlert';
 import useOrganization from 'sentry/utils/useOrganization';
+import useProjectSdkNeedsUpdate from 'sentry/utils/useProjectSdkNeedsUpdate';
 
 const LOCAL_STORAGE_KEY = 'replay-canvas-supported';
 
 /**
  * Displays a notice about canvas support if a <canvas> element is detected in the replay, but not recorded
  */
-export function CanvasSupportNotice() {
+export function CanvasSupportNotice({projectId}: {projectId: string | undefined}) {
```

Comment:
we can also get the project in this function since we already use `useReplayContext` to get the replay!

Reply:
ooh good catch!